recipe/meta.yaml

```diff
@@ -49,38 +64,30 @@ outputs:
     script: install-python.sh
     build:
       activate_in_script: True
+      string: "{{ fft_impl }}_py{{ py }}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"
```

Comment:
Add
```
    ignore_run_exports:
       - blas
```
here

Reply:
This is to stop the blas `run_exports` coming from `defaults::mkl_devel`

Reply:
@isuruf, I don't follow this statement, can you elaborate a little?

Reply:
Is this just because we don't want to ping to the MKL build of blas when our package doesn't technically require it?

Reply:
Yes. you need MKL for the FFT and not for BLAS.